fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialogBottomSheet.kt

```diff
@@ -0,0 +1,220 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.Divider
+import org.mozilla.fenix.compose.SwitchWithLabel
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Firefox Translation options bottom sheet dialog.
+ */
+@Composable
+fun TranslationOptionsDialogBottomSheet(
+    translationOptionsList: List<TranslationSwitchItem>,
+    onBackClicked: () -> Unit,
+    onTranslationSettingsClicked: () -> Unit,
+    aboutTranslationClicked: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .background(
+                color = FirefoxTheme.colors.layer2,
+                shape = RoundedCornerShape(topStart = 8.dp, topEnd = 8.dp),
+            ),
+    ) {
+        TranslationOptionsDialogHeader(onBackClicked)
+        LazyColumn {
+            items(translationOptionsList) { item: TranslationSwitchItem ->
+                SwitchWithLabel(
+                    checked = item.isEnabled,
+                    onCheckedChange = item.stateChange,
+                    label = item.textLabel,
+                    modifier = Modifier
+                        .padding(start = 72.dp, end = 16.dp)
+                        .clearAndSetSemantics {
+                            contentDescription = item.contentDescription
+                        },
+                )
+                if (item.hasDivider) {
+                    Divider(Modifier.padding(top = 4.dp, bottom = 4.dp))
+                }
+            }
+        }
+        Text(
```

Comment:
As a rule of thumb, could you add a new line in between each of your composable as we have done in other composables? This will help make things easier to read.
```suggestion
        TranslationOptionsDialogHeader(onBackClicked)

        LazyColumn {
            items(translationOptionsList) { item: TranslationSwitchItem ->
                SwitchWithLabel(
                    checked = item.isEnabled,
                    onCheckedChange = item.stateChange,
                    label = item.textLabel,
                    modifier = Modifier
                        .padding(start = 72.dp, end = 16.dp)
                        .clearAndSetSemantics {
                            contentDescription = item.contentDescription
                        },
                )

                if (item.hasDivider) {
                    Divider(Modifier.padding(top = 4.dp, bottom = 4.dp))
                }
            }
        }

        Text(
```
Snippet of some of the suggested changes, but please self-review your other changes and help improve readability of your code.

Reply:
Done